Dockerfile

```diff
@@ -7,16 +7,16 @@ RUN apk add --no-cache git
 RUN go get ./...
 
 WORKDIR /go/src/github.com/hacdias/filemanager/cmd/filemanager
-RUN go build -ldflags "-X main.version=$(git tag -l --points-at HEAD)"
+RUN CGO_ENABLED=0 go build -a -ldflags "-X main.version=$(git tag -l --points-at HEAD)"
```

Comment:
This is really nice! Thanks! Could you please just change it to `go build` please? We don't need the ldflags anymore 😄